Sources/DatadogObjc/OpenTracing/OTGlobal+objc.swift

```diff
@@ -7,19 +7,38 @@
 import Foundation
 import Datadog
 
+@objcMembers
+public class DDGlobal: NSObject {
+    public internal(set) static var tracer: OTTracer = noopTracer {
+        didSet {
+            // We must also set the Swift `Global.tracer`
+            // as it's used internally by auto-instrumentation feature.
+            if let ddTracer = tracer.dd?.swiftTracer {
+                Global.sharedTracer = ddTracer
+            }
+        }
+    }
+    public internal(set) static var rum = DDRUM(isNoOp: true) {
```

Comment:
`DDRUM(isNoOp: true)` looks very weird. Why not just providing no-op implementation in `DatadogObjc` as we did for tracer (`let noopTracer: OTTracer = DDNoopTracer()`)?

Reply:
(replied in the comment above ☝️ )